app/models/main_page_refresher.rb

```diff
@@ -0,0 +1,28 @@
+module MainPageRefresher
+  def prepare_data(transaction)
+    organization = transaction.organization
+    transaction.bank_account.touch
```

Comment:
@dissident fix in current. And it's not a bug. It's just wrong written code. In this method you dont make any save so nothing to recalculate. If you make any updated in other method and then call this method then it's wrong. This touch should be where update is done.

Fix it here.